libs/UIHelpers/transfer_function_widget.cpp

```diff
@@ -410,26 +431,44 @@ void TransferFunctionWidget::drawLegend() {
   glLoadMatrixf(projection);
   glMatrixMode(GL_MODELVIEW);
   glLoadMatrixf(modelview);
+}
 
-
-
+void TransferFunctionWidget::draw_legend(float legend_pos_x, float legend_pos_y, float legend_width, float legend_height)
+{
+  bool show_legend = true;
+  ImGui::SetNextWindowPos(ImVec2(legend_pos_x, legend_pos_y));
+  ImGui::SetNextWindowSize(ImVec2(legend_width, legend_height));
+  ImGui::Begin("##legend", &show_legend, ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoScrollbar | ImGuiWindowFlags_NoResize | ImGuiWindowFlags_NoTitleBar);
+  ImGuiIO &io = ImGui::GetIO();
+  ImTextureID my_tex_id = io.Fonts->TexID;
+  float my_tex_w = (float)io.Fonts->TexWidth;
+  float my_tex_h = (float)io.Fonts->TexHeight;
```

Comment:
should these variables be prepended with `m_` instead of `my_`

Reply:
This is code from a third party library. https://github.com/brown-ccv/VR-Volumeviewer/issues/49